dqc_us_rules/dqc_us_0041.py

```diff
@@ -0,0 +1,294 @@
+# (c) Copyright 2016, XBRL US Inc. All rights reserved.
+# See license.md for license information.
+# See PatentNotice.md for patent infringement notice.
+from .util import messages
+
+import json
+import os
+import re
+import time
+
+from arelle.ModelDtsObject import ModelConcept
+from arelle import XbrlConst, ModelXbrl
+from arelle.FileSource import openFileStream, saveFile, openFileSource
+
+
+_CODE_NAME = 'DQC.US.0041'
+_RULE_VERSION = '1.1'
+
+
+ugtDocs = (
+    {"year": 2014,
+     "namespace": "http://fasb.org/us-gaap/2014-01-31",
+     "docLB": "http://xbrl.fasb.org/us-gaap/2014/elts/us-gaap-doc-2014-01-31.xml",  # noqa
+     "entryXsd": "http://xbrl.fasb.org/us-gaap/2014/entire/us-gaap-entryPoint-std-2014-01-31.xsd",  # noqa
+     },
```

Comment:
SG: Multiline alignment, recommend: 
```
    {
        "year": 2014,
        "namespace": "http://fasb.org/us-gaap/2014-01-31",
        "docLB": "http://xbrl.fasb.org/us-gaap/2014/elts/us-gaap-doc-2014-01-31.xml",  # noqa
        "entryXsd": "http://xbrl.fasb.org/us-gaap/2014/entire/us-gaap-entryPoint-std-2014-01-31.xsd",  # noqa
    },
```
And replicating that down for the other entries